### ISSUE TITLE
8399537

### ISSUE THREAD

**Issue by dasa376** [GoogleChrome/chrome-extensions-samples#601]
#!/bin/bash cd ~/repos/mdn/content/ yarn start

alias mdn-server="open -a Terminal.app ~/bin/mdn-start.bsh"

alias wpt-server="open -a Terminal.app ~/bin/wpt-serve.bsh"

#!/bin/bash cd ~/repos/wpt/ ./wpt serve

div[aria-label="Image"]::before { 	content: "WARNING: no useful ALT text"; 	background: yellow; 	border: 0.5em solid red; 	border-radius: 1em; 	box-shadow: 0 0 0.5em black; 	…

div[aria-label="Image"] { 	filter: grayscale(1) contrast(0.5); } div[aria-label="Image"]:hover { 	filter: none; } 

div[aria-label="Image"], div[aria-label="Embedded video"] { 	filter: grayscale(1) contrast(0.5); } div[aria-label="Image"]:hover, div[aria-label="Embedded video"]:hover { 	filter: none; }

testing GIF This Is ATest Fool GIF Corona Test GIF by euronews Test Fail GIF Corona Virus GIF by guardian Its ATest Josh Subdquist GIF Corona Stay Home GIF by INTO ACTION Is This A Test GIF Stressed Out Community GIF A1b2c3 GIF

alias please='sudo "$BASH" -c "$(history -p !!)"'

please() { 	if [ "$1" ]; then 		sudo $@ 	else 		sudo "$BASH" -c "$(history -p !!)" 	fi }

function add_contact_links ( $text ) {   if (is_feed()) {       $text .= '       <hr><p><a href="' . get_permalink($post) . '#commentform">Add a comment to the post, or <a href="mailto:admin@example.com?subject=In%20reply%20to%20%22' . str_replace(' ', '%20', get_the_title()) . '%22">email a reply</a>.</p>';   }   return $text; } add_filter('the_content', 'add_contact_links');

function add_contact_links ( $text ) {   $text .= '   <hr><p><a href="' . get_permalink($post) . '#commentform">Add a comment to the post, or <a href="mailto:admin@example.com?subject=In%20reply%20to%20%22' . str_replace(' ', '%20', get_the_title()) . '%22">email a reply</a>.</p>';   return $text; } add_filter('the_content_feed', 'add_contact_links');
